### PR TITLE
Add a metadata parameter to enable/disable a tutorial in the topic page

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

I added a metadata parameter to be able to hide a tutorial in its topic page: it can not then be accessed via the topic page (only via its full URL).

Bérénice

### CODE REVIEW

**Comment by bgruening** [comment]
Can we make enable true the default option to not being forced to write it?

**Comment by bebatut** [comment]
Yes we can do that. Do I keep it in the metadata.yaml files?

**Comment by bgruening** [comment]
I would remove it. This is very special and people should not worry about it.

**Comment by bebatut** [comment]
@bgruening I removed the `enable` for most tutorial's metadata, except the ones to disable

**Comment by bgruening** [comment]
Thanks looks good.
@bebatut can you rebase.

**Comment by bebatut** [comment]
@bgruening Rebased!

**Comment by martenson** [comment]
@bebatut is this removing all links to that tutorial from the training site? If so we should be careful not to inhibit crawlers' indexing efforts and maybe link from somewhere else to it.

**Comment by bebatut** [comment]
@martenson The tutorial page will be still generated, but not indexed in the topic page (with other tutorials for the topic)

File: topics/assembly/metadata.yaml [hunk 1]
```diff
@@ -24,10 +24,10 @@ material:
 -
     title: "Introduction to Genome Assembly"
     type: "tutorial"
+    enable: "true"
```
Comment (bgruening) [inline topics/assembly/metadata.yaml]: can be removed

File: topics/admin/metadata.yaml [hunk 2]
```diff
@@ -8,6 +8,7 @@ material:
 -
     title: "Introduction"
     type: "introduction"
+    enable: "true"
```
Comment (bgruening) [inline topics/admin/metadata.yaml]: can be removed

**Comment by bgruening** [comment]
Thanks @bebatut !

**Pull request merged**